README.md

```diff
@@ -28,18 +27,31 @@ This simple step will already configure the loader overlay for use.
 
 After that configuration you can just run the command:
 
-```
-context.showLoaderOverlay()
+```dart
+context.loaderOverlay.show()
 ```
 
 This will show the overlay with the default loading indicator. The default loading configured is to just show a centered CircularProgressIndicator
 
 To hide the overlay (after the async processing, for example), just run the command:
 
+```dart
+context.loaderOverlay.hide()
 ```
-context.hideLoaderOverlay()
+
+You can check if overlay is visible:
+
+```dart
+final isVisible = context.loaderOverlay.visible
 ```
 
+And get the type of overlay widget:
```

Comment:
> And get the type of overlay widget:

Change to `And get the type of overlay widget that you pass (if so) to `context.loaderOverlay.show`